src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,34 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class StringTest {
+    @Test
+    void split() {
+        String[] result = "1,2".split(",");
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @Test
+    @DisplayName("특정 위치의 문자를 성공적으로 반환한다.")
+    void IndexTest() {
```

Comment:
메서드의 이름은 소문자로 시작하도록 작성해보면 어떨까요?
```suggestion
    void indexTest() {
```